internal/bminventory/inventory.go

```diff
@@ -166,7 +171,16 @@ const ignitionConfigFormat = `{
 			"name": "root"
 		},
 		"contents": { "source": "data:,{{.PULL_SECRET}}" }
-	  }{{if .RH_ROOT_CA}},
+	},
+	{
+		"overwrite": true,
+		"path": "/root/assisted.te",
+		"mode": 420,
+		"user": {
+			"name": "root"
+		},
+		"contents": { "source": "data:text/plain;charset=utf-8,module%20assisted%201.0%3B%0D%0Arequire%20%7B%0D%0A%20%20%20%20%20%20%20%20type%20chronyd_t%3B%0D%0A%20%20%20%20%20%20%20%20type%20container_file_t%3B%0D%0A%20%20%20%20%20%20%20%20type%20spc_t%3B%0D%0A%20%20%20%20%20%20%20%20class%20unix_dgram_socket%20sendto%3B%0D%0A%20%20%20%20%20%20%20%20class%20dir%20search%3B%0D%0A%20%20%20%20%20%20%20%20class%20sock_file%20write%3B%0D%0A%7D%0D%0A%23%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%20chronyd_t%20%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%0D%0Aallow%20chronyd_t%20container_file_t%3Adir%20search%3B%0D%0Aallow%20chronyd_t%20container_file_t%3Asock_file%20write%3B%0D%0Aallow%20chronyd_t%20spc_t%3Aunix_dgram_socket%20sendto%3B" }
```

Comment:
Would it be better to base64 this content?

Reply:
I thought it would be a little bit more readable as a data URL scheme. But I guess there is no difference. :)

Reply:
I don't think it's going to be readable in any scheme and it's easier to base64 something than to uri escape it in my opinion.

Reply:
A more readable option is to have a var for the text and pass the b64.Encoded result as an argument to the template.Execute